database/count.sh

```diff
@@ -0,0 +1,17 @@
+source .env
+export PGPASSWORD=$DB_PASSWORD
+
```

Comment:
Nice script for getting an understanding of what our database is holding at any given time, without having to look manually. LGTM.